lib/views/activites/photograph_activity_view.dart

```diff
@@ -104,13 +105,14 @@ class _PhotographActivityViewState extends State<PhotographActivityView> {
               ? Column(
                   children: <Widget>[
                     BodyText(
-                       "Your photo:",
+                      "Your photo:",
                     ),
                     Padding(
                       padding: const EdgeInsets.only(top: 20.0),
                       child: Container(
-                        height: (MediaQuery.of(context).size.height / 100) * 45.2,
-                        width: (MediaQuery.of(context).size.width / 4) * 3.5,
+                        height:
+                            (MediaQuery.of(context).size.height / 100) * 45.2,
+                        width: Screen.width(context, percentage: 12.5),
```

Comment:
Your math is wrong.

 1 / 4 * 3.5 = 87.5 - not 12.5

You'll need to go through and check all of your math.

Reply:
Will do i wasn't sure which way around these are supposed to be